respy/tests/utils.py

```diff
@@ -1,12 +1,12 @@
 import numpy as np
 
-from respy.shared import get_example_model
+from respy import get_example_model
```

Comment:
Wouldn't a `from respy.interface import get_example_model` be more suitable.

Reply:
In general, I am always a little bit undecided whether the test suite should use the official interface where possible or import from the destination of function definition. E.g. numpy also uses ``np.array`` throughout the implementation. For respy, we seem to have used the official interface where possible. So, it should be ```import respy as rp``. Good catch!